### PR TITLE
Max & min values of range filter to public

### PR DESCRIPTION
**Pull request by kardi31**
cc @shieldo

### CODE REVIEW

**Comment by shieldo** [comment]
Looks pretty good - given these are new public methods, could you add tests for the implementation?

**Comment by kardi31** [comment]
@shieldo Added a change to `FacetField` to make `getFilter` method public. Added tests for both.

**Comment by shieldo** [comment]
@kardi31 Can you sort out the failing test?

**Comment by kardi31** [comment]
That's fixed now @shieldo

File: Facet/FacetField.php [hunk 1]
```diff
@@ -45,7 +45,7 @@ public function __toString()
     /**
      * @return AttributeInterface
      **/
-    private function getFilter()
+    public function getFilter()
```
Comment (shieldo) [inline Facet/FacetField.php]: Why has this been made public? Given that the implementation delegates most of its methods down to the filter, it shouldn't be necessary to expose a method that isn't on the interface that this class adheres to (and shouldn't be).

Comment (shieldo) [reply Facet/FacetField.php]: OK I have an understanding of why, but don't agree with it. We shouldn't mutate visibility for this kind of reason, unless it's a design decision with appropriate changes to interfaces etc.

**Comment by kardi31** [comment]
@shieldo Removed changes to `getFilter` method as it's not needed anymore.

**Pull request merged**